### ISSUE TITLE
Twitter button violates WCAG 4.1.2

### ISSUE THREAD

**Issue by auchenberg** [microsoft/vscode-docs#2618]
aria-hidden-focus: ARIA hidden element must not contain focusable elements (1) WCAG 4.1.2

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI
- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

Code path:
```
<a href="" aria-hidden="true" onclick="followOnTwitter()" role="button" tabindex="0">Follow @code</a>
--

**Comment by Nikhitha-13** [comment]
Note:
The issue is observed through out the application in the following URLs:
1. code.visualstudio.com/docs
2. code.visualstudio.com/download.
3. code.visualstudio.com/docs/supporting/faq.
4. code.visualstudio.com/docs/setup/Linux
5. code.visualstudio.com/docs/getstarted/settings
6. code.visualstudio.com/docs/python/python-tutorial
7. code.visualstudio.com/updates/v1_30
8. code.visualstudio.com/docs/editor/whyvscode.
9.  code.visualstudio.com/docs/getstarted/tips-and-tricks
10. code.visualstudio.com/updates/v1_29
11. code.visualstudio.com/docs/editor/integrated-terminal
12. code.visualstudio.com/docs/editor/debugging
13. code.visualstudio.com/docs/editor/versioncontrol
![Bug_#2618_(4 1 2)](https://user-images.githubusercontent.com/50437925/57447724-51c33b00-7275-11e9-838e-10196f439bf8.png)
![Bug_#2618_(4 1 2)issue](https://user-images.githubusercontent.com/50437925/57447784-70c1cd00-7275-11e9-9ccb-4befdcdb6808.png)
![Bug_#2618_issue](https://user-images.githubusercontent.com/50437925/57447807-7f0fe900-7275-11e9-8416-f20a3cb117a5.png)

**Comment by Nikhitha-13** [comment]
![Bug_#2618_Issue](https://user-images.githubusercontent.com/50437925/57448033-137a4b80-7276-11e9-95f0-a5afedff44ac.png)
![Bug_#2618-issue](https://user-images.githubusercontent.com/50437925/57448080-29880c00-7276-11e9-96b3-5c4f9f8bc809.png)
![Bug_#2618_Issue](https://user-images.githubusercontent.com/50437925/57448118-3d337280-7276-11e9-978f-d7d3f24ddf69.png)
![Bug_#2618_issue(4 1 2)](https://user-images.githubusercontent.com/50437925/57448151-50464280-7276-11e9-90d4-bccb323e7209.png)
![Bug_#2618_issue](https://user-images.githubusercontent.com/50437925/57448178-618f4f00-7276-11e9-8845-f3626371ac92.png)
![Bug-#2618_issue](https://user-images.githubusercontent.com/50437925/57448220-7966d300-7276-11e9-8cfb-33c08f74c1c9.png)
![Bug-#2618_issue](https://user-images.githubusercontent.com/50437925/57448261-8d123980-7276-11e9-95a8-ba92f5151ee8.png)

**Comment by Nikhitha-13** [comment]
![Bug_2618_issue(4 1 2)](https://user-images.githubusercontent.com/50437925/57448326-bc28ab00-7276-11e9-8e2e-e52294180d6b.png)
![Bug_#2618_issue](https://user-images.githubusercontent.com/50437925/57448360-cba7f400-7276-11e9-92b8-cf01d6c9c0cc.png)
![Bug_#2618-issue](https://user-images.githubusercontent.com/50437925/57448384-debac400-7276-11e9-99fa-5669af829620.png)
![Bug_#2618_issue(4 1 2)](https://user-images.githubusercontent.com/50437925/57448516-2fcab800-7277-11e9-9762-5cea53a39d1e.png)

**Comment by auchenberg** [comment]
Fixed in https://github.com/microsoft/vscode-website/commit/1bd4cd36a5642f4005b12058ad1bd44861e83b03

**Comment by Nikhitha-13** [comment]
Ran keros and the issue is now not found.
